index.html

```diff
@@ -1,7 +1,8 @@
 <!doctype html>
 <html>
   <head>
-    <link rel='stylesheet' href='styles.css'>
+    <link rel='stylesheet' type="text/css" href='/Users/lizafloyd/wdi/exercises/checkpoint-01/styles.css'>
```

Comment:
:+1:

While this link would work locally, however, if you were to deploy this the stylesheet would not load because you are referencing it using an absolute path that is unique to your local environment.